### DISCUSSION TITLE
Audible converted file metadata is blank and cannot be changed

### DISCUSSION THREAD

**Discussion by csmith1510** [audiamus/AaxAudioConverter#179] [Q&A]
I use BookLibConnect to download, decrypt, and export the audiobook to an aax file. From there I load the file and convert to an m4b file, however I am unable to edit the Details tab of the book properties where it says the title, author, etc. It is all blank. This is only happening on two books from the same author. The rest of his books as well as others work fine. I have verified I have full permissions to the file as well. Anyone run into this? I tried to search first, but came up empty.

**Comment by audiamus** [comment]
It could be specific to these books. AAX Audio Converter primarily looks for embedded meta data. You could check what's there with the original .m4b file created by Book Lib Connect using an external tool like [MediaInfo](https://mediaarea.net/en/MediaInfo).
